### ISSUE TITLE
CubeData object

### ISSUE THREAD

**Issue by gunbra32** [CAB-LAB/cablab-core#18]
Currently the method get() of the CubveData class returns a masked numpy array with the data of the requested variable only. It does not, however, return the respective variable data of the dimensions, i.e. of time, lat, and lon. I think, it would be better if get() would instead return an object that contains the data of the varible itself, the dimensional variables, and even a land-sea-mask, or a country mask. This would ensure that a non-expert user could not easily mess up the dimensions. 

For example: 
```python
albedo = your_cube_reader.get("Albedo", None, (0,10), (10,20))
```
would then give the user an object with data, lat,lon, and whatever you want to put there, even some basic methods: 

```python
albedo.data
albedo.lat
albedo.lon
albedo.country
albedo.plot_map(time=anytime)
albedo.slice([t_0,t_n],[lat_0,lat_n],[lon_0, lon_n])

**Comment by meggart** [comment]
I agree, this would make a lot of sense for the plotting API. What would you suggest when reading multiple variables at once? At the moment, a Dictionary is returned, can we leave it that way and do:
````python
    values = your_cube_reader.get(("Albedo","Emission"), None, (0,10), (10,20))

    values.lon
    values.lat
    values["Albedo"]
````
or would you prefer values.data["Albedo"]?

**Comment by forman** [comment]
I suggest making it self-similar:

````python

values = your_cube_reader.get(("Albedo","Emission"), None, (0,10), (10,20))

values.lon
values.lat
values.country
values.plot_map(time=anytime)  # multi-variable plot

albedo = values["Albedo"]

albedo.data
albedo.lat
albedo.lon
albedo.country
albedo.plot_map(time=anytime) # plot of 'Albedo'
````

**Comment by meggart** [comment]
>I suggest making it self-similar:

Even better! :sunglasses:

**Comment by forman** [comment]
Issue would be solved or addressed, if we use xarray (http://xarray.pydata.org) to represent Cube objects.

**Comment by forman** [comment]
Fixed by #29
